audacity/audacity
# 7647 effect panel part 2

Resolves: #7647 




Recommended:

## QA:
Here is the [figma spec](https://www.figma.com/design/w2kAh1gKKV5bu7htY7158W/08---Effects?node-id=1-4&node-type=canvas&t=rgnwM1ZZEvSANEgM-0). Not every aspect of this spec is implemented yet. Here is a list of what is known _not_ to be yet implemented. It is meant to be exhaustive, so if you find something not implemented that's not in this list, please raise it.
* master effects aren't there yet
* power buttons are yet ineffective
* the effect panel can yet only be opened via the track header
* the effect panel can yet only be closed via the effect panel header
* the "Get more effects" option is not there yet
* the UI window of effects isn't yet summonable, neither when adding an effect nor when clicking on the button with the effect's name (basically you're stuck with the effect's default parameters)
* at the moment, the "No effect" behaviour is the same as in AU3, (i.e., there isn't the "Delete effect" option yet)
* reordering of effects isn't yet implemented
* there is no way of scrolling the effect list when it gets taller than the available space

Aside this, everything should be there.
Things to test not part of the spec:
- [x] effects can be added, removed or replaced during playback

Known issues that can be addressed later:
* changing a track's name isn't reflected in the effect panel automatically
